x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/asset_criticality_file_uploader/hooks.ts

```diff
@@ -72,7 +72,8 @@ export const useFileValidation = ({ onError, onComplete }: UseFileChangeCbParams
         },
       });
 
-      const parserConfig: ParseConfig = {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This matches the previous behavior where the type of `ParseConfig` defined `ParsedResult.data` as `Array<any>`